pkg/auth/auth.go

```diff
@@ -72,6 +72,8 @@ type SpecialAuthURLs struct {
 	RequestToken string
 	// KubeAdminLogout is the logout URL for the special kube:admin user in OpenShift.
 	KubeAdminLogout string
+	// Issuer is the base URL for the OAuth server
```

Comment:
This makes it easier to build other urls, but perhaps we simply want to put `OAuthHealth` endpoint here instead.